backend/core/MegaParse/megaparse/config.py

```diff
@@ -22,6 +22,6 @@ def from_yaml(cls, file_path: str):
 
 
 class MegaparseConfig(MegaparseBaseConfig):
-    strategy: str = "fast"
+    strategy: str = "auto"
```

Comment:
Need to check the impact of this on the worker

Reply:
@chloedia This was set for a reason in megaparse ? The OCR took too much time ?

Reply:
The heuristic I set up in previous project was to check the area of the "image" vs text in the page. 
Parsers like fitz return the the bbox of image, I guess unstructured should return that. Nonnative text in pdfs is represented as an image. a threshold of 0.8 worked but it depends heavily on the document being processed. 
Another heuristic is the existence of glyph encoding in the pdf. We should test that.

Reply:
Yes normally it already fallbacks to auto when fast is empty (according to unstructured) that s why I used it !